CHANGELOG

```diff
@@ -1,3 +1,7 @@
+# Version 1.8.0
```

Comment:
I assume you mean `major` change. Also, should I leave the changelog alone like the `version.rb` below? Just keep the actual code changes?